packages/braid-design-system/src/lib/components/MenuRenderer/MenuRenderer.tsx

```diff
@@ -220,6 +270,18 @@ export const MenuRenderer = ({
     }
   };
 
+  useEffect(() => {
+    const handleResize = () => {
+      dispatch({ type: WINDOW_RESIZE });
+    };
+
+    if (open) {
+      window.addEventListener('resize', handleResize);
+    } else {
+      window.removeEventListener('resize', handleResize);
+    }
```

Comment:
The recommended way to remove global handlers is in the clean up function of the effect. The clean up function is run when the component unmounts, or before runs again on state change.

So something like:
```suggestion
    if (open) {
      window.addEventListener('resize', handleResize);
    }
    
    return () => window.removeEventListener('resize', handleResize);
```